remote/startup.sh

```diff
@@ -16,6 +16,7 @@ else
 fi
 
 python3 /opt/setup_workspace.py;
+stat /opt/setup_workspace_purevariants.py &>/dev/null && python3 /opt/setup_workspace_purevariants.py
```

Comment:
```suggestion
python3 /opt/setup_workspace_purevariants.py
```

What's the reason of the `stat` command?

Reply:
The `startup.sh` file is in the remote so it should not be dependant on if the file provided by the base image exists, hence the `stat`.